tarantool_test.go

```diff
@@ -81,6 +81,7 @@ func BenchmarkClientSerial(b *testing.B) {
 		b.Errorf("No connection available")
 	}
 
+	b.ResetTimer()
```

Comment:
It seems weird that in the first commit you're writing "broken" (= without timer reset) tests and fixing them up together with older tests in the second one. It's better to reorder them.

Reply:
Then maybe we should rewrite "Follows up #122" to "Part of #122"

Reply:
Fixed
```
commit 16062755ab41a38595de732e29cbf352d03dfbae
Author: Rianov Viacheslav <slavarianov@yandex.ru>
Date:   Thu Apr 28 14:40:28 2022 +0300

    test: fix incorrect perf measurement
    
    Added calls of b.ResetTimer() to all Benchmark tests before
    running bench loop. That call clears all counters of allocs,
    timers etc. The preparation before bench loop affected perf results.
    
    Part of #122
```